auth/src/main/kotlin/com/hedvig/android/auth/NetworkAuthRepository.kt

```diff
@@ -37,6 +37,8 @@ class NetworkAuthRepository(
     val requestBody = buildJsonObject {
       put("method", loginMethod.name)
       put("country", market)
+      put("callbackSuccess", "https://google.se?q=success")
+      put("callbackFailure", "https://google.se?q=failure")
```

Comment:
So from what I understand, we also need to edit our ZignSec webview to match these. As a first step it could be just changing the "failure" -> "fail"
https://github.com/HedvigInsurance/android/blob/4074740bd0cb72fdb37693c492ea55ceb377116c/app/src/main/kotlin/com/hedvig/app/feature/zignsec/ui/ZignSecWebViewFragment.kt#L47
But in reality we could even match the entire URL so we have full control over this, instead of matching just part of the URL. But regardless, the "failure" and "fail" need to match is my understanding there.